app/src/main/java/co/nimblehq/compose/crypto/ui/screens/information/SellBuyButton.kt

```diff
@@ -0,0 +1,57 @@
+package co.nimblehq.compose.crypto.ui.screens.information
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.ui.theme.Color.FireOpal
+import co.nimblehq.compose.crypto.ui.theme.Color.White
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp0
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp12
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp16
+import co.nimblehq.compose.crypto.ui.theme.Style
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun SellBuyButton(
```

Comment:
- Should we name this button to be a common button, such as `CustomButton`? 😛 
- Should we move it into `co.nimblehq.compose.crypto.ui.common`?

Reply:
@luongvo Let's move it later 🙏 For now it uses only on the `Detail` screen.